_workinggroups/quantumcomputing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Quantum Computing"
+layout: plain
+---
+
+# The HSF Quantum Computing working group
+
+## Introduction
+
+The Quantum Computing Working Group fosters collaboration among the HEP experiments and institutes to exploit the possibilities offered by quantum computing.
+
+Quantum computing is a very active research field and prototype computers, simulators, and toolkits are now available. The participants in the working group share experiences, ideas, know-how, demonstrators, code snippets, and documents with the aim of fostering common research and development in this field, for the benefit of the HEP community.
+
+
+[Mandate](/organization/working-group-mandates.html).
```

Comment:
It seems a bit off to reference the mandate page, which was written more specifically with Sim, Reco and Analysis groups in mind. I think the activities is generally good, and we should probably factorise that for all WGs, but the specific page referenced is a bit too CWPish, where we didn't have QC at the time.